lib/SendGridCompatibility/Email.js

```diff
@@ -5,8 +5,9 @@
  *
  * @param options object that contains initial values
  */
-function Email(options){
-  for (var option in options) {
```

Comment:
Truthfully, all that SendGridCompatibility is going away. But I'll remember that. Now that we are only supporting node 4+, we can do all the ES6 stuff.